lisa/trace.py

```diff
@@ -303,7 +303,7 @@ def __init__(self,
                  trace_path,
                  plat_info=None,
                  events=None,
-                 normalize_time=True,
+                 normalize_time=False,
```

Comment:
Assuming we don't have traces with more than ~10 days of uptime that should be alright. After that, the nanosecond information will become noise.

Reply:
The thing I'm trying to "fix" is somewhat orthogonal to this; the thing is I can easily re-load a `TestBundle` and print some of its internal data to figure out what happened. The issue is that the internal `Trace` is loaded with the default `normalize_time` (thus `True`) and that isn't easily hijackable.

So I would print some df, get normalized timestamps, and wouldn't be able to compare this with kernelshark. Other than the long uptime issue, AFAIR the main reason for normalization was for "pretty plots". In my experience this has been more of an annoyance (my bread and butter being kernelshark), so I'd like to change that.

Reply:
We can change the default if we really don't expect anything more than 10 days uptime (and we could add a warning for non normalized time in that case), but it's pretty easy to get a trace with the parameter you want:
```
trace = test_bundle.get_trace(events=['foo', 'bar'], normalize_time=False)
```
`test_bundle.trace` is meant to be suited to test methods more than anything else, conflicting requirements can either be met with `get_trace()` or by introducing a new property (maybe a more friendly `test_bundle.debug_trace` that would parse all events, skip trace_window and provide non-normalized timestamps ?)